prepline_general/api/general.py

```diff
@@ -190,6 +189,13 @@ def partition_pdf_splits(
     return results
 
 
+logger = logging.getLogger("unstructured_api")
+handler = logging.StreamHandler()
+formatter = logging.Formatter("%(asctime)s %(processName)-10s %(levelname)-8s %(message)s")
+handler.setFormatter(formatter)
+logger.addHandler(handler)
```

Comment:
we set up logger in logger_config.yaml instead of inside codes, and those lines have default setting

Reply:
What is the appropriate way to map the logger in the code? I'd like to have some logging statements that can associate with the log level (i.e. debug).

Reply:
you could just remove those lines i think? in `logger_config.yaml`
```
unstructured_api:
    level: DEBUG
    handlers:
      - standard_handler
    propagate: no
```
`standard_handler` already have the formatter setting, and you could also create your own